### PR TITLE
test

### PR DESCRIPTION
**Pull request by coderjonny**
this is testing out a pull request. demo-ing.

### CODE REVIEW

**Comment by coderjonny** [comment]
@iteratelance can you check out?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Demo
 
-- Travis.ci test
```
Comment (iteratelance) [inline README.md:3]: sucks ass

Comment (coderjonny) [inline README.md:3]: oh rly?

**Comment by iteratelance** [comment]
Looks great, good job. Fixes #5

**Comment by iteratelance** [comment]
close pull request.

**Pull request merged**